build_dist.sh

```diff
@@ -28,7 +28,7 @@ EOF
 	exit 0
 fi
 
-tags=""
+tags="${TAGS:-}"
```

Comment:
This is error prone... if they use `--extra-small` and some build tags, we'll still add our own omit build tags.

Reply:
added a check in flag parsing below to prohibit setting both flags and $TAGS